apps/src/code-studio/pd/application/teacher/TeacherApplicationConstants.js

```diff
@@ -16,7 +16,7 @@ const styles = {
   },
   questionText: {
     fontSize: 14,
-    lineHeight: 20,
+    lineHeight: '20px',
```

Comment:
Well, that's annoying ... fontSize comes through as px but lineHeight comes through as em. I think I might change what the style guide recommends and use strings from now on.

Reply:
good point! did some double checking...
line-height is a weird property that by default uses unitless values. You can set it by px or em, but best practice is to actually set as a ratio of the font size, so I changed it to that. (the 20 was setting it to 20 times the font size)

Reply:
Ahh, I see ... just happened to look the same as 20em ... well, I won't rush to change the style guide, but I'm still not loving how unclear it is. Anyways, saw you changed it––looks good!